.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python:
-  - "2.7"
-  - "3.6"
+  - 2.7
+  - 3.6
```

Comment:
The problem with this change is that "3.6" is a string while 3.6 is a float.  This works fine when you only want to specify the major and minor Python version can break down when you want to specify the micro version as well.  The issue here is that 3.6.3 is a valid version of Python but it is not a valid floating point number.